<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADTool</class>
 <widget class="QDialog" name="ADTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="acclist_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>   Danh sách Acc  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="acclist_textbox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkacc_button">
       <property name="text">
        <string>Check Account</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="defaultpwd_label">
       <property name="text">
        <string>Default Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="defaultpwd_textbox">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>12345abcd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="date_label">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="date_textbox">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="targetou_label">
       <property name="text">
        <string>Target OU</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="targetou_textbox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="acctable_widget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Acc</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DN</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>✔</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="log_widget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>192</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="resetpwd_button">
       <property name="text">
        <string>Reset Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disable_button">
       <property name="text">
        <string>Disable Account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unlock_button">
       <property name="text">
        <string>Unlock</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
